css_tasks/task11/index.html

```diff
@@ -0,0 +1,896 @@
+          <div>
+            <h4>Seo</h4>
+            <p>
+              Ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.
+            </p>
+          </div>
+        </div>
+        <div class="services__element">
+          <span><img src="assets/home@2x.png" alt=""/></span>
+          <div>
+            <h4>Css/Html</h4>
+            <p>
+              Lorem dolor sit amet, consectetur adipiscing elit, sed do tempor.
+            </p>
+          </div>
+        </div>
+        <div class="services__element">
+          <span><img src="assets/image@2x.png" alt=""/></span>
+          <div>
+            <h4>Digital</h4>
+            <p>Sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- DESIGN -->
+    <section class="design">
+      <picture>
+        <source
+          media="(min-width:679px) and (max-width:1024px)"
+          srcset="assets/layer-45@2x-min.jpg"
+        />
+        <img
+          src="assets/layer-45@3x-min.jpg"
+          alt="Mountains"
+          class="home-background__image"
+        />
+      </picture>
+      <header class="main-header main-header--design">
+        <h2 class="main-header__headline">For all devices</h2>
+        <p class="main-header__title">unique design</p>
+        <div class="main-header__separator"></div>
+      </header>
+      <picture>
+        <source
+          media="(min-width:679px) and (max-width:1024px)"
+          srcset="assets/website@2x.png"
+        />
+        <img src="assets/website@3x.png" alt="Website" class="design__image" />
+      </picture>
+      <picture>
+        <source
+          media="(min-width:679px) and (max-width:1024px)"
+          srcset="assets/white@2x.png"
+        />
+        <img
+          src="assets/white@3x.png"
+          alt="Mobile Phone"
+          class="design__image design__image--mobile"
+        />
+      </picture>
+    </section>
+    <!-- SERVICE -->
+    <section class="container">
+      <header class="main-header">
+        <h2 class="main-header__headline">Service</h2>
+        <p class="main-header__title">what we do</p>
+        <div class="main-header__separator"></div>
+        <p class="main-header__text">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ex ea commodo consequat.
+        </p>
+      </header>
+      <div class="service">
+        <div class="service__element">
+          <picture>
+            <source
+              media="(max-width:1024px)"
+              srcset="assets/layer-43@2x.jpg"
+            />
+            <img src="assets/layer-43@3x.jpg" alt="Man painting the wall" />
+          </picture>
+        </div>
+        <div class="service__element">
+          <details>
```

Comment:
when this details is opened it should take same size as image and don't change size of all section(it can be left for js part for this task)

Reply:
OK